hub/constants.py

```diff
@@ -1,4 +1,17 @@
 BYTE_PADDING = b"\0"
+
+# number of bytes per unit
+KB = 1024
+MB = 1024 * KB
+GB = 1024 * MB
```

Comment:
[super-nitpick] these are [MiB](https://en.wikipedia.org/wiki/Byte#Multiple-byte_units), not [MB](https://en.wikipedia.org/wiki/Megabyte)

Reply:
not a nitpick, just accurate

Reply:
i was also confused